solutions/dedicated/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0, <1.6.0"
+  required_providers {
+    ibm = {
+      source = "IBM-Cloud/ibm"
+      # Use "greater than or equal to" range in modules
```

Comment:
remove this code comment - this is not a module, its a DA, so we lock into exact version like you have